pool_automation/configure.yml

```diff
@@ -0,0 +1,16 @@
+---
+- name: Configure pool
+  gather_facts: false
+  hosts: nodes
+  roles:
+    - indy_node
+  tags: [ 'pool', 'nodes' ]
```

Comment:
Do we really need `pool` tag here?

Reply:
I think it would be handy to be able to refer that using `pool` since the playbook actually configures not a set of separate nodes but the pool of them.